stac_fastapi/types/stac_fastapi/types/rfc3339.py

```diff
@@ -122,6 +122,9 @@ def str_to_interval(interval: Optional[str]) -> Optional[DateTimeType]:
             detail="Interval string contains more than one forward slash.",
         )
 
+    if len(values) == 1:
+        values = [values[0], values[0]]
+
```

Comment:
This is to make sure GET (attr class) and POST (stac-pydantic model) requests will return the same results